app/gui/view/graph-editor/src/component/node/action_bar/icon.rs

```diff
@@ -160,3 +160,51 @@ pub mod skip {
         }
     }
 }
+
+/// Icon for the button to disable re-evaluation. Looks like a crossed-out arrow loop.
+pub mod disable_reevaluation {
+    use super::*;
+
+    use std::f32::consts::FRAC_PI_2;
+    use std::f32::consts::FRAC_PI_6;
+
+    ensogl::shape! {
+        (style: Style, color_rgba: Vector4<f32>) {
+            let fill_color  = Var::<color::Rgba>::from(color_rgba);
+            let width        = Var::<Pixels>::from("input_size.x");
+            let height       = Var::<Pixels>::from("input_size.y");
```

Comment:
You will see many places with aligned `=` but this rule is outdated. Macros are not auto-formatted, and we haven't managed to update all of them manually. But in new code, do not align `=` in assignment statements.